packages/app-builder/src/models/custom-list.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  type CreateCustomListBody as CreateCustomListBodyDto,
+  type CustomList as CustomListDto,
+  type CustomListValue as CustomListValueDto,
+  type CustomListWithValues as CustomListWithValuesDto,
+  type UpdateCustomListBody as UpdateCustomListBodyDto,
```

Comment:
You can directly rename those type in the API, so they stick to the backend name (`dto.Type`) and give an easy way to find non-refactored dto types from the OAS file.

In short, you need to rename the type in the OAS, generate the API and you're good to go

Reply:
👍  makes sense